### ISSUE TITLE
📝 formatter not formatting embedded snippets

### ISSUE THREAD

**Issue by dyc3** [biomejs/biome#10420]
### Environment information

<details>

```bash

playground

```

</details>

### Configuration

```JSON

```

### Playground link

https://biomejs.dev/playground/?quoteStyle=single&trailingCommas=none&indentScriptAndStyle=true&tab=formatter&pane=Diagnostics&code=ewB7ACAAJAB0ACgAIgByAG8AbwBtAC4AbgBlAHgAdAAtAHYAaQBkAGUAbwAtAHYAbwB0AGUAIgApACAAfQB9AAoA&language=vue

in this playground, quote style is set to `single`. the expected output should change the double quotes to single quotes.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
@dyc3 I don't think that's the issue. The formatter isn't working for the snippets: https://biomejs.dev/playground/?quoteStyle=single&jsxQuoteStyle=single&trailingCommas=none&indentScriptAndStyle=true&tab=formatter&pane=Diagnostics&code=ewB7ACAAJAB0ACgACgAgACAACgAgACAAIgByAG8AbwBtAC4AbgBlAHgAdAAtAHYAaQBkAGUAbwAtAHYAbwB0AGUAIgAKAAoACgApACAAfQB9AAoA&language=vue

Can you create a CLI reproduction?

**Comment by dyc3** [comment]
yes, it happens on the cli too https://github.com/dyc3/biome-repro-1779364433271

**Comment by ematipico** [comment]
That's weird. How come tests are passing?

**Comment by dyc3** [comment]
Turns out we just aren't marking those snippets for formatting, according to my clanker at least. I'll post a fix after I fix the preview releases

**Comment by dyc3** [comment]
Nevermind, this is more complicated than I was expecting.
